code/obj/item/clothing/gimmick.dm

```diff
@@ -933,6 +933,14 @@
 	icon_state = "sneakmans"
 	item_state = "sneakmans"
 
+/obj/item/clothing/suit/armor/sneaking_suit/costume
+	desc = "On closer inspection this is a cheap cosplay outfit with an obvious zipper."
+
+	setupProperties()
+		..()
+		setProperty("meleeprot", 2)
+		setProperty("rangedprot", 0)
```

Comment:
Maybe use delproperty instead so that it won't show "rangedprot: 0" on the tooltip etc